examples/employee-app/src/main/java/io/helidon/service/employee/ReposFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.service.employee;
+
+public class ReposFactory {
+
+    public static EmployeeRepository getRepo(String driverType) {
```

Comment:
You can move this code as a static method in the `EmployeeRepository`, this removes a class and simplifies the code.

i.e instead of doing `ReposFactory.getRepo`, you would do something like `EmployeeRepository.create(driverType)`.

Reply:
Done.